### PR TITLE
chore: restore welcome screen

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-9576

### CODE REVIEW

File: src/components/Menus/ThemeSubMenu/useThemeMenuContent.tsx [hunk 1]
```diff
@@ -33,7 +36,15 @@ export const useThemeMenuContent = () => {
         [TrackingEventParameter.SwitchedTemplate]: theme,
       },
     });
+    if (!(theme === 'system' || theme === 'light' || theme === 'dark')) {
```
Comment (oktapodia) [inline src/components/Menus/ThemeSubMenu/useThemeMenuContent.tsx:39]: better to use an include to for readability

File: src/components/Widgets/Widget.style.ts [hunk 2]
```diff
@@ -13,15 +13,98 @@ export const WidgetWrapper = styled(Box, {
...
-    '&:hover': {
-      marginTop: 0,
-    },
-  }),
   zIndex: 2,
-  // [theme.breakpoints.up('sm' as Breakpoint)]: {
-  //   maxWidth: 416,
-  // },
+
+  '> div': {
```
Comment (oktapodia) [inline src/components/Widgets/Widget.style.ts:18]: Not fan of this as it is only related to welcomeScreen, can you add it to the welcomescreen component itself?

File: src/hooks/useWelcomeScreen.ts [hunk 3]
```diff
@@ -1,25 +1,42 @@
...
+    () =>
+      activeTheme === 'default' ||
+      activeTheme === 'light' ||
+      activeTheme === 'dark' ||
+      activeTheme === 'system' ||
+      cookie.theme === 'default' ||
+      cookie.theme === 'light' ||
+      cookie.theme === 'dark' ||
+      cookie.theme === 'system' ||
+      !cookie.theme,
```
Comment (oktapodia) [inline src/hooks/useWelcomeScreen.ts:36]: Would be better to simplify it

```suggestion
() => {
  const validThemes = ['default', 'light', 'dark', 'system'];
  return validThemes.includes(activeTheme) || validThemes.includes(cookie.theme) || !cookie.theme;
},
```

File: src/hooks/useWelcomeScreen.ts [hunk 4]
```diff
@@ -29,6 +46,9 @@ export const useWelcomeScreen = (
   }, [cookie.welcomeScreenClosed, isMultisigSigner]);
 
   const updateState = (closed: boolean) => {
+    if (!enabled) {
```
Comment (oktapodia) [inline src/hooks/useWelcomeScreen.ts:49]: Is this check needed?

File: src/components/Menus/ThemeSubMenu/useThemeMenuContent.tsx (hunk 1 above)
Comment (dennyscode) [reply src/components/Menus/ThemeSubMenu/useThemeMenuContent.tsx:39]: Good idea!

File: src/hooks/useWelcomeScreen.ts (hunk 4 above)
Comment (dennyscode) [reply src/hooks/useWelcomeScreen.ts:43]: It´s meant to ensure that state changes on the welcome-screen are only enabled if the theme supports the welcome-screen.

File: src/components/Widgets/Widget.style.ts (hunk 2 above)
Comment (dennyscode) [reply src/components/Widgets/Widget.style.ts:18]: I don´t think we can move this easily to the WelcomeScreen itself as they´re not directly related in the tree as children/parents.

This ugly hack allows to directly style the widget via the widget-wrapper. It´s mainly needed for positioning.
If you know of a cleaner way, I´d love to hear. Maybe the "container" prop within the widgetTheme is a better choice??

**Review by admsk2 — APPROVED** [review]

**Review by tcheee — APPROVED** [review]

**Pull request merged**